Cargo.toml

```diff
@@ -5,19 +5,28 @@ members = [
     "webb-gadget",
     "zk-gadget",
     "test-gadget",
-    "playground"
+    "playground",
+    "protocols/mp-ecdsa"
 ]
 
 [workspace.dependencies]
 gadget-core = { path = "./gadget-core" }
 webb-gadget = { path = "./webb-gadget" }
 zk-gadget = { path = "./zk-gadget" }
 zk-playground = { path = "./playground" }
+mp-ecdsa-protocol = { path = "./protocols/mp-ecdsa" }
+
+dkg-runtime-primitives = { git = "https://github.com/webb-tools/dkg-substrate" }
```

Comment:
Would prefer you don't bring in any of these crates @tbraun96. You should copy over only what needs to be brought over from there.